core/templates/dev/head/pages/exploration_editor/simple_editor_tab/questions/TextEditorDirective.js

```diff
@@ -0,0 +1,115 @@
+// Copyright 2017 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Directive for a text interaction question in the simple editor.
+ */
+
+oppia.directive('textEditor', [
+    'QuestionIdService', 'AnswerGroupObjectFactory', 'RuleObjectFactory',
+    'StatusObjectFactory', 'OutcomeObjectFactory', 'UrlInterpolationService',
+    function(QuestionIdService, AnswerGroupObjectFactory, RuleObjectFactory,
+          StatusObjectFactory, OutcomeObjectFactory, UrlInterpolationService) {
```

Comment:
Break previous line after '(', and indent the arg list by 4 spaces from line 22.